visitor/TreeVisitor.cpp

```diff
@@ -122,6 +122,13 @@ void TreeVisitor::handleUnion(clang::RecordDecl *record, std::string name) {
 void TreeVisitor::handleStruct(clang::RecordDecl *record, std::string name) {
     std::string newName = "struct_" + name;
 
+    if (record->hasAttr<clang::PackedAttr>()) {
+        llvm::errs() << "Warning: struct " << name << " is packed. "
+                     << "Packed structs are not supported by Scala Native. "
+                     << "Access to fields will not work correctly.\n";
```

Comment:
I suggest we start documenting this in a user's guide. Do you have a preference for a documentation tool? Else we can go with Paradox.